compiler/rustc_attr/src/builtin.rs

```diff
@@ -1036,21 +1036,21 @@ pub fn allow_internal_unstable<'a>(
     sess: &'a Session,
     attrs: &'a [Attribute],
 ) -> Option<impl Iterator<Item = Symbol> + 'a> {
```

Comment:
```suggestion
) -> impl Iterator<Item = Symbol> + 'a {
```

Reply:
Are you sure about this? These are `pub`lic functions.